scripts/php/process.php

```diff
@@ -211,6 +212,12 @@ private function buildOptOutRequestor () {
     return $requestor;
   }
 
+  private function buildOptOutDateTimeString () {
```

Comment:
could this be just a generic function we could use to add a timestamp to any email?